server/UserRegistration.java

```diff
@@ -0,0 +1,467 @@
+package newbank.server;
+
+import com.sun.tools.javac.Main;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Scanner;
+import java.util.*;
+import java.sql.SQLException;   // If have an error thrown from execution.
+
+public class UserRegistration extends Main {
```

Comment:
`Main` is deprecated and shouldn't be used. Please remove this and implement a `main` method in the `UserRegistration` class